region/lowernorfair/west/Screw Attack Room.json

```diff
@@ -968,45 +1456,207 @@
         "leaveWithGrappleTeleport": {
           "blockPositions": [[2, 34]]
         }
-      }
+      }
+    },
+    {
+      "id": 38,
+      "link": [3, 2],
+      "name": "Reset Fall Speed (Wave)",
+      "requires": [
+        {"obstaclesCleared": ["A"]},
+        {"doorUnlockedAtNode": 2},
+        "canResetFallSpeed",
+        "Wave",
+        {"heatFrames": 200}
+      ],
+      "unlocksDoors": [
+        {
+          "types": ["ammo"],
+          "requires": [
+            "never"
+          ]
+        }
+      ]
+    },
+    {
+      "id": 39,
+      "link": [3, 2],
+      "name": "Reset Fall Speed (Quick Shot)",
```

Comment:
This works well with comeInShinecharging strats.

Reply:
I tried it the other day but wasn't able to make it onto the door ledge that way. I assume I'm missing something

Reply:
Oh of course, that makes sense. Added a strat now for this. I found it can work well even if you don't slide off the ledge: https://videos.maprando.com/video/1767

When I was trying it before, I was doing it wrong, trying to spin jump and then mid-air morph-unmorph.